test/e2e/instascale_machinepool_test.go

```diff
@@ -50,9 +50,9 @@ func TestInstascaleMachinePool(t *testing.T) {
 	defer connection.Close()
 
 	// check existing cluster machine pool resources
-	// look for machine pool with aw name - expect not to find it
+	// look for a machine pool with a label key equal to aw name - expect NOT to find it
 	test.Expect(GetMachinePools(test, connection)).
-		ShouldNot(ContainElement(WithTransform(MachinePoolId, Equal("test-instascale-g4dn-xlarge"))))
+		ShouldNot(ContainElement(WithTransform(MachinePoolLabels, HaveKey("test-instascale"))))
```

Comment:
I've been working on the nodepools test and had to update this as it's checking for the entire label which includes `test-instascale` plus the instance type. This needs to be updated in 3 places in the test.
```suggestion
		ShouldNot(ContainElement(WithTransform(MachinePoolLabels, HaveKey(HavePrefix("test-instascale")))))
```